package.json

```diff
@@ -1,61 +1,81 @@
 {
   "name": "eslint-loader",
   "version": "2.2.1",
-  "description": "eslint loader (for webpack)",
-  "keywords": [
-    "lint",
-    "linter",
-    "eslint",
-    "loader",
-    "webpack"
-  ],
-  "repository": "https://github.com/webpack-contrib/eslint-loader.git",
-  "author": "Maxime Thirouin",
+  "description": "Eslint loader for webpack",
   "license": "MIT",
+  "repository": "webpack-contrib/eslint-loader",
+  "author": "Maxime Thirouin",
+  "homepage": "https://github.com/webpack-contrib/eslint-loader",
+  "bugs": "https://github.com/webpack-contrib/eslint-loader/issues",
+  "main": "dist/cjs.js",
+  "engines": {
+    "node": ">= 8.9.0"
+  },
+  "scripts": {
+    "start": "npm run build -- -w",
+    "prebuild": "npm run clean",
+    "build": "cross-env NODE_ENV=production babel src -d dist --ignore \"src/**/*.test.js\" --copy-files",
+    "clean": "del-cli dist",
+    "commitlint": "commitlint --from=master",
+    "lint:prettier": "prettier \"{**/*,*}.{js,json,md,yml,css}\" --list-different",
+    "lint:js": "eslint --cache src test",
+    "lint": "npm-run-all -l -p \"lint:**\"",
+    "prepare": "npm run build",
+    "release": "standard-version",
+    "security": "npm audit",
+    "test:only": "cross-env NODE_ENV=test jest",
+    "test:watch": "cross-env NODE_ENV=test jest --watch",
+    "test:coverage": "cross-env NODE_ENV=test jest --collectCoverageFrom=\"src/**/*.js\" --coverage",
+    "pretest": "npm run lint",
+    "test": "cross-env NODE_ENV=test npm run test:coverage",
+    "defaults": "webpack-defaults"
+  },
   "files": [
-    "index.js"
+    "dist"
   ],
   "peerDependencies": {
     "eslint": ">=1.6.0 <7.0.0",
```

Comment:
I think we can drop old eslint version, let's keep only 5 and 6

Reply:
lol, I'm doing this now